examples/IRServer/IRServer.ino

```diff
@@ -68,6 +68,7 @@ void handleRoot() {
                 "<head><title>" HOSTNAME " Demo </title>" \
                 "<meta http-equiv=\"Content-Type\" " \
                     "content=\"text/html;charset=utf-8\">" \
+                "<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0\">" \
```

Comment:
Linter error:
`examples/IRServer/IRServer.ino:71:  Lines should be <= 80 characters long  [whitespace/line_length] [2]`